CMakeLists.txt

```diff
@@ -1029,6 +1033,27 @@ if(EMSCRIPTEN)
 endif()
 ###
 
+### mal as dll
+add_library(
+  libcb SHARED
```

Comment:
What's the implication? Does it mean `cbl.exe` is not a standalone anymore and will have the mal dll aside?

Reply:
I wanted to get "cargo test" running.. WIP :D